guides/common/modules/proc_repository-sets.adoc

```diff
@@ -0,0 +1,17 @@
+[id="Repository_Sets_{context}"]
+= Repository Sets in {Project}
+
+Repository sets show available repositories to each host. If your host is subscribed to a particular Content View and Lifecycle environment, the default repository set is limited to the repositories that are specifically available to that host.
```

Comment:
```suggestion
Repository sets show available repositories to each host.
If your host is subscribed to a particular Content View and Lifecycle environment, the default repository set is limited to the repositories that are specifically available to that host.
```

Reply:
@bangelic what's happening here is that we have a rule about writing one sentence per line.